ui.R

```diff
@@ -53,6 +40,21 @@ shinyUI(
         # tabPanel("Output", verbatimTextOutput("temp"))
       )
     ),
+    tabPanel(
```

Comment:
why did this need to move?

Reply:
by moving it, when we click on the link directly Graphic Explorer opens up , instead of Data Explorer it is a quick fix in the next commit after adding docker, I will change it back to normal and we will think of a better way